configs/projects/puppet-agent.rb

```diff
@@ -211,8 +211,8 @@
   if platform.is_huaweios?
     proj.component "rubygem-net-scp"
     proj.component "rubygem-mini_portile2"
-    proj.component "rubygem-nokogiri"
     proj.component "rubygem-net-netconf"
+    proj.component "rubygem-nokogiri"
```

Comment:
I'm not sure the load order actually matters here. I believe Vanagon will actually build a graph and map out dependencies that way, so this may be superfluous.